src/app/(auth)/login/page.tsx

```diff
@@ -17,7 +17,7 @@ export default function Login() {
                             <Image
                                 className="mr-2 w-80"
                                 src="./images/logo.svg"
-                                loading="eager"
+                                priority
```

Comment:
Cool, this will not make the page slower to first render right? I can see a PNG affecting the first loading time if we not lazy it

Reply:
you are quite right, new commit to address that